android/src/main/java/com/reactnativejitsimeet/RNJitsiMeetViewManager.java

```diff
@@ -36,45 +52,129 @@ public String getName() {
     public RNJitsiMeetView createViewInstance(ThemedReactContext context) {
         if (mJitsiMeetViewReference.getJitsiMeetView() == null) {
             RNJitsiMeetView view = new RNJitsiMeetView(context.getCurrentActivity());
-            view.setListener(this);
             mJitsiMeetViewReference.setJitsiMeetView(view);
         }
+        registerForBroadcastMessages();
         return mJitsiMeetViewReference.getJitsiMeetView();
     }
 
-    public void onConferenceJoined(Map<String, Object> data) {
-        WritableMap event = Arguments.createMap();
-        event.putString("url", (String) data.get("url"));
-        mReactContext.getJSModule(RCTEventEmitter.class).receiveEvent(
-                mJitsiMeetViewReference.getJitsiMeetView().getId(),
-                "conferenceJoined",
-                event);
+    @ReactProp(name="options")
```

Comment:
I took this from https://github.com/sajanthomas01/react-native-jitsi-meet-extended
I removed part of the code to match the way we currently handle on iOS, but we can add later.
Sadly, it creates fragmentation on the way to have a call, but I couldn't find the reason why `@ReactMethod call` isn't working https://github.com/RocketChat/react-native-jitsi-meet/blob/427873beb6fcbd7c32dd9019550fb313cede292f/android/src/main/java/com/reactnativejitsimeet/RNJitsiMeetModule.java#L35